src/identity.js

```diff
@@ -101,7 +101,14 @@ export class Identity extends EventEmitter {
         this._sessionInitiatedSent = false;
         this.window = window;
         this.clientId = clientId;
-        this.cache = new Cache(this.window && this.window.localStorage);
+        const cache = false;
```

Comment:
You made this const and then reassign it below. I guess you can either make it a let, or just drop this variable and have the original `new Cache..` in the try and another with no argument in the catch (don’t think the localStorage.test = 2 looks particularly elegant tbh)